docs/UI/Alert/assets/AlertLocalWebApp.txt

```diff
@@ -0,0 +1,19 @@
+participant Local Web App
+participant SDL
+participant HMI
+
+note over Local Web App, HMI:<align:center>**Preconditions:**\n  1.Web app (WEB_VIEW) is registered and activated on the HMI\n 2. HMI displays the WEB_VIEW template
+
+Local Web App->SDL: Alert (alertText)
+SDL->HMI: UI.Alert (alertStrings, appID)
+HMI->SDL: UI.OnSystemContext (systemContext=ALERT,appID)
+
+note over HMI: <align:center>The Alert pop-up is displayed on HMI\n overlaying WEB_VIEW area
+
+note over HMI: <align:center>User closes the Alert pop-up\n**Touch event first goes through the POP-UP area**
+
+HMI->SDL: Buttons.OnButtonEvent
+HMI-->SDL: UI.Alert:(SUCCESS)
+HMI -> SDL: UI.OnSystemContext(systemContext=MAIN,appID)
```

Comment:
Could you please add the response arrow at the end going to the app?

Reply:
@kshala-ford, thank you, please, see fixed in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/219/commits/efdae1bf4ec782ce807b778c03dd6b8d9f7ace48